codecparsers/vc1Parser.cpp

```diff
@@ -748,6 +782,16 @@ namespace VC1 {
             }
         }
 
+        if (m_frameHdr.picture_type == FRAME_B) {
```

Comment:
this is duplicate with L965 is it ok to wrapper it as function, such as decodeBFraction?